forte/data/entry_type_generator.py

```diff
@@ -39,3 +52,39 @@ def get_type_attributes():
 
         """
         return _get_type_attributes()
+
+    @staticmethod
+    def get_entry_attributes_by_class(input_entry_class_name: str):
+        """Get type attributes by class name. input_entry_class_name should be
+        an object class dotted name path.
+        Please note must add "data class" decorator to all the class and their
+        ancestors.
+
+
+        Args:
+            input_entry_class_name: An object class dotted name path.
+
+        Returns:
+             A list of attributes corresponding to the input class.
```

Comment:
The documentation didn't mention what would happen if the input class is decorated by `dataclass`